lib/document-group/index.js

```diff
@@ -106,13 +81,13 @@ class DocumentGroup {
 
   /**
    * Creates document group with specified documents or templates.
-   * At least one document or template must contain fields. Documents should not be signed, contain epending invites or signature requests.
+   * At least one document or template must contain fields. Documents should not contain pending invites.
    * @param {DocumentGroupCreateParams} data - create document group payload
    * @param {function(err: ApiErrorResponse, res: DocumentGroupCreateResponse)} [callback] - error first node.js callback
    */
   static create ({
     token,
-    ids,
+    document_ids: ids,
```

Comment:
`ids` was put there intentionally because documents may be of different types (template of document) and method has been already published with this signature. Do not change this!